packages/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -86,9 +86,7 @@ export class Arc extends React.Component<IArcProps, IArcState> {
 
   private _hoverOn(data: IChartDataPoint, mouseEvent: React.MouseEvent<SVGPathElement>): void {
     mouseEvent.persist();
-    if (this.props.activeArc === this.props.data!.data.legend || this.props.activeArc === '') {
```

Comment:
Why did you remove these checks

Reply:
These checks were not completely removed but used in the setState function call to update the visibility of the callout